mathesar_ui/src/sections/table-view/Header.svelte

```diff
@@ -99,52 +139,83 @@
       width:{columnPosition.get(column.name).width}px;
       left:{columnPosition.get(column.name).left + paddingLeft}px;">
       <span class="type">
-        {#if column.type === 'INTEGER'}
-          #
-        {:else if column.type === 'VARCHAR'}
-          T
-        {:else}
-          i
-        {/if}
+        {determineDataIcon(column.type)}
       </span>
       <span class="name">{column.name}</span>
 
-      <Dropdown closeOnInnerClick={true}
-                triggerClass="opts" triggerAppearance="plain"
+      <Dropdown isOpen={dropdownStates[column.name]} triggerClass="opts" triggerAppearance="plain"
```

Comment:
Since we are going to manual trigger, it would be cleaner to create a separate Cell component for the column headers, and maintain a single dropdown state within each of them.